.github/workflows/release.yml

```
@@ -117,48 +152,36 @@ jobs:
             args: ''
 
     runs-on: ${{ matrix.platform }}
-    # 兜底：apt/网络偶发卡死时避免占用 runner 到 GitHub 默认的 6 小时上限
     timeout-minutes: 60
 
     steps:
-      - name: Check if this platform should run
+      - name: Filter target platform
         id: check
         shell: bash
         run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:6:30: Double quote to prevent globbing and word splitting

(shellcheck)